examples/jsm/nodes/display/NormalMapNode.js

```diff
@@ -1,6 +1,7 @@
 import TempNode from '../core/TempNode.js';
 import ModelNode from '../accessors/ModelNode.js';
-import { ShaderNode, positionView, normalView, uv, join, cond, add, sub, mul, dFdx, dFdy, cross, max, dot, normalize, inversesqrt, equal } from '../ShaderNode.js';
+import { ShaderNode } from '../utils/ShaderNodeUtils.js';
```

Comment:
Can you move `ShaderNode` to core? For we can use `Nodes.ShaderNode` too.

Reply:
We can use it currently too, because ShaderNode is exported from ShaderNode.js, and therefore is also exported from Nodes.js.

Reply:
Okk, let's just move the file then. like `../core/ShaderNode.js`

Reply:
Done.